app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,36 @@
+window.onload = init();
+
+function init(){
+  document.getElementById("upvote").addEventListener("click", upVote);
+  document.getElementById("downvote").addEventListener("click", downVote);
+}
+
+function upVote() {
+  console.log("upvote clicked");
```

Comment:
Remove `console.log` here, that's the equivalent of a `puts` in your production code